File: modules/models/app/br/com/republicavirtual/CepServiceVO.java
package br.com.republicavirtual;


import com.thoughtworks.xstream.annotations.XStreamAlias;
import play.api.libs.json.*;
import scala.Function1;

import java.util.Map;

@XStreamAlias("webservicecep")
public class CepServiceVO  implements Writes<CepServiceVO>
{

    @XStreamAlias("uf")
    private String uf;

    @XStreamAlias("cidade")
    private String cidade;

    @XStreamAlias("bairro")
    private String bairro;

    @XStreamAlias("tipo_logradouro")
    private String tipo_logradouro;

    @XStreamAlias("logradouro")
    private String logradouro;

    @XStreamAlias("resultado")
    private String resultado;

    @XStreamAlias("resultado_txt")
    private String resultado_txt;

    public String getBairro() {
        return bairro;
    }

    public void setBairro(String bairro) {
        this.bairro = bairro;
    }

    public String getCidade() {
        return cidade;
    }

    public void setCidade(String cidade) {
        this.cidade = cidade;
    }

    public String getLogradouro() {
        return logradouro;
    }

    public void setLogradouro(String logradouro) {
        this.logradouro = logradouro;
    }

    public String getResultado() {
        return resultado;
    }

    public void setResultado(String resultado) {
        this.resultado = resultado;
    }

    public String getResultado_txt() {
        return resultado_txt;
    }

    public void setResultado_txt(String resultado_txt) {
        this.resultado_txt = resultado_txt;
    }

    public String getTipo_logradouro() {
        return tipo_logradouro;
    }

    public void setTipo_logradouro(String tipo_logradouro) {
        this.tipo_logradouro = tipo_logradouro;
    }

    public String getUf() {
        return uf;
    }

    public void setUf(String uf) {
        this.uf = uf;
    }


    /**
     * Convert the object into a JsValue
     *
     * @param o
     */
    @Override
    public JsValue writes(CepServiceVO o) {
        return null;
    }

    /**
     * transforms the resulting JsValue using transformer function
     *
     * @param transformer
     */
    @Override
    public Writes<CepServiceVO> transform(Function1<JsValue, JsValue> transformer) {
        return null;
    }

    /**
     * transforms resulting JsValue using Writes[JsValue]
     *
     * @param transformer
     */
    @Override
    public Writes<CepServiceVO> transform(Writes<JsValue> transformer) {
        return null;
    }
}

File: modules/models/app/br/com/republicavirtual/CepServiceVO.scala
package br.com.republicavirtual

import _root_.util.xDevSerialize
import com.thoughtworks.xstream.annotations.XStreamAlias
import play.api.libs.json._
import scala.Function1
import com.fasterxml.jackson.databind.annotation.JsonSerialize
import util._
import java.util.Map

@JsonSerialize
@XStreamAlias("webservicecep")
class CepServiceVO extends xDevSerialize {
  def getBairro: String = {
    return bairro
  }

  def setBairro(bairro: String) {
    this.bairro = bairro
  }

  def getCidade: String = {
    return cidade
  }

  def setCidade(cidade: String) {
    this.cidade = cidade
  }

  def getLogradouro: String = {
    return logradouro
  }

  def setLogradouro(logradouro: String) {
    this.logradouro = logradouro
  }

  def getResultado: String = {
    return resultado
  }

  def setResultado(resultado: String) {
    this.resultado = resultado
  }

  def getResultado_txt: String = {
    return resultado_txt
  }

  def setResultado_txt(resultado_txt: String) {
    this.resultado_txt = resultado_txt
  }

  def getTipo_logradouro: String = {
    return tipo_logradouro
  }

  def setTipo_logradouro(tipo_logradouro: String) {
    this.tipo_logradouro = tipo_logradouro
  }

  def getUf: String = {
    return uf
  }

  def setUf(uf: String) {
    this.uf = uf
  }

  def serialize(): JsObject = {
    Json.obj(
      "uf" -> this.getUf,
      "cidade" -> this.getCidade,
      "bairro" -> this.getBairro,
      "tipo_logradouro" -> this.getTipo_logradouro,
      "logradouro" -> this.getLogradouro,
      "resultado" -> this.getResultado,
      "resultado_txt" -> this.getResultado_txt
    )
  }

  @XStreamAlias("uf") private var uf: String = null
  @XStreamAlias("cidade") private var cidade: String = null
  @XStreamAlias("bairro") private var bairro: String = null
  @XStreamAlias("tipo_logradouro") private var tipo_logradouro: String = null
  @XStreamAlias("logradouro") private var logradouro: String = null
  @XStreamAlias("resultado") private var resultado: String = null
  @XStreamAlias("resultado_txt") private var resultado_txt: String = null
}

